epsagon/trace.py

```diff
@@ -126,6 +128,7 @@ def initialize(
         :param url_patterns_to_ignore: URL patterns to ignore in HTTP data
          collection.
         :param keys_to_ignore: List of keys to ignore while extracting metadata.
```

Comment:
which one is stronger?

Reply:
Ignored keys. added to readme and switched running order of `remove_ignored_keys` and `get_dict_with_allow_keys` functions